### DISCUSSION TITLE
Info on builds

### DISCUSSION THREAD

**Discussion by machinaeZER0** [diegolix29/shadPS4#14] [Q&A] [answered]
Hello! It's been amazing to see all the work being done on the emulator lately, and the builds on this fork are particularly interesting to me. Apologies if this is laid out somewhere that I'm missing, but since multiple builds seem to be pushed over here I wanted to clarify what each of them is testing - I believe PartBB is testing adding new particle effects, but I'm also seeing "release" builds and "PGO" builds on the release page. Would it be possible to gain more insight about these individual builds?

Thank you so much for your work!

**Comment by diegolix29** [comment, marked as answer]
Hi sorry i didnt reply here lol i didnt knew it was on. Yeah now its all changed, Full souls, is for the other games  from fromsotfware, it has hacks for 3 games so its kinda unstable. Now mainbb has the particles  and PRTBB are pr testing branch, it dosenthave the particles in it cause it may conflict with other prs.

**Reply by machinaeZER0** [reply]
Ahh, thank you so much! Last question, just to clarify - when we download a "release" build (release-shadPS4-2025-02-07-f2a4e45, for example), are those mainbb builds? Or for mainbb builds would we want to be downloading from the Actions tab?

**Reply by diegolix29** [reply]
yeah the actions goes go the release page, the release one is full souls branch to play other souls games. the prtbb is a pull request testing branch, and prerelease is mainbb, they all autoupdate into themselfves and can switch between them.

**Reply by machinaeZER0** [reply]
Ahh, perfect! So for Bloodborne specifically I'll go back to pulling mainbb from the actions tab, rather than using the release builds (since those are adding hacks not needed for this specific game, if I'm understanding you correctly). Thanks again for taking the time to explain!

**Comment by gsanjin** [comment]
Honestly, it is so confusing to follow so many of your branches when builds don't have differentiation in file downloads name, especially since there is no any explanation on your main page (which is just c/p from main shadPS4). Maybe more info could be put in github front page and branch name could be part of build package name. I see PRTBB branch builds does have that in build name (for example: PRTBB-shadPS4-2025-02-08-72440f3), although I have no idea what PRTBB is and what's difference from other branches :) ?

**Reply by diegolix29** [reply]
Hi those suggestions are more complex to add tho, cause i dont make the releases on my own they are pushed through code, so its pretty much github making the names for the downloads. The main page as its a fork dosent have much stuff that i want to change as i dont want to be a independant fork, so i keep it as main shad for that sake.

**Reply by gsanjin** [reply]
Thank you for explanations Diegolix29 as I am not fully used to github processes and keep up the good work!
